Testscripts/Windows/PERF-STORAGE-FIO-HB.ps1

```diff
@@ -0,0 +1,316 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+<#
+.Synopsis
+	Perform a simple VM hibernation in Azure
+	This feature might be available in kernel 5.7 or later. By the time,
+	customized kernel will be built.
+	# Hibernation will be supported in the general purpose VM with max 16G vRAM
+	# and the GPU VMs with max 112G vRAM.
+
+.Description
+	1. Prepare swap space for hibernation
+	2. Compile a new kernel (optional)
+	3. Update the grup.cfg with resume=UUID=xxxx where is from blkid swap disk
+	4. Run the first fio testing
+	5. Hibernate the VM, and verify the VM status
+	5. Resume the VM and verify the VM status.
+	6. Verify no kernel panic or call trace
+	7. Run the second fio testing.
+	8. Verify IOPS counts
```

Comment:
you didn't verify IOPS in ps1 script

Reply:
IOPS or other measurements can be implemented in the next PR. I remove that comment for now.

Reply:
Will update later. 5f02ff780c9a63ca11f5fe9c2b6aaa6be88ef158